core/src/jvm_bridge/batch_iterator.rs

```diff
@@ -35,7 +34,7 @@ impl<'a> CometBatchIterator<'a> {
 
     pub fn new(env: &mut JNIEnv<'a>) -> JniResult<CometBatchIterator<'a>> {
         // Get the global class reference
-        let class = get_global_jclass(env, Self::JVM_CLASS)?;
+        let class = env.find_class(Self::JVM_CLASS)?;
```

Comment:
Ok since all CI tests are passed. I think previous approach is for the older jni_rs APIs. I remember the APIs requires static reference previously.